pallets/invulnerables/src/lib.rs

```diff
@@ -171,10 +220,10 @@ pub mod pallet {
             T::UpdateOrigin::ensure_origin(origin)?;
 
             <Invulnerables<T>>::try_mutate(|invulnerables| -> DispatchResult {
-                match invulnerables.binary_search(&who) {
-                    Ok(_) => return Err(Error::<T>::AlreadyInvulnerable)?,
-                    Err(pos) => invulnerables
-                        .try_insert(pos, who.clone())
+                match invulnerables.contains(&who) {
+                    true => return Err(Error::<T>::AlreadyInvulnerable)?,
+                    false => invulnerables
```

Comment:
Use if/else

Reply:
does that make any difference?

Reply:
It makes the code easier to read for me, if you're asking about performance it is the same.

Reply:
can we keep it for the sake of diversity? :)

Reply:
the majority won, done. :)